uk_bin_collection/uk_bin_collection/councils/NorthTynesideCouncil.py

```
@@ -1,138 +1,77 @@
         check_uprn(user_uprn)
-        check_postcode(user_postcode)
 
+        # diable warnings so that we can ignore cert verification
         requests.packages.urllib3.disable_warnings()
-        s = requests.Session()
 
-        # Get the first form
-        response = s.get(
-            "https://my.northtyneside.gov.uk/category/81/bin-collection-dates",
+        # Fetch the schedule page
+        response = requests.get(
```

Ruff (0.14.0)

`requests` may be undefined, or defined from star imports

(F405)